test/contrib/redshift_test.py

```diff
@@ -44,10 +43,42 @@ class DummyS3CopyToTable(luigi.contrib.redshift.S3CopyToTable):
     aws_access_key_id = AWS_ACCESS_KEY
     aws_secret_access_key = AWS_SECRET_KEY
     copy_options = ''
+    prune_table = ''
+    prune_column = ''
+    prune_date = ''
 
     def s3_load_path(self):
         return 's3://%s/%s' % (BUCKET, KEY)
 
+class DummyS3CopyToTempTable(luigi.contrib.redshift.S3CopyToTable):
+    # Class attributes taken from `DummyPostgresImporter` in
+    # `../postgres_test.py`.
+    host = 'dummy_host'
+    database = 'dummy_database'
+    user = 'dummy_user'
+    password = 'dummy_password'
+    table = luigi.Parameter(default='stage_dummy_table')
+    columns = (
+        ('some_text', 'text'),
+        ('some_int', 'int'),
+    )
+
+    aws_access_key_id = AWS_ACCESS_KEY
+    aws_secret_access_key = AWS_SECRET_KEY
+    copy_options = ''
+    sql = ["insert into dummy_table select * from stage_dummy_table;"]
+    prune_date = 'current_date - 30'
+    prune_column = 'dumb_date'
```

Comment:
@Tarrasch If the `prune_*` variables should be marked as properties, shouldn't `table_type` and `table_attributes` also be marked as such?